setup/aix61/manualBootstrap.md

```diff
@@ -98,6 +121,13 @@ LIBPATH=/usr/lib curl https://bootstrap.pypa.io/get-pip.py | python
 ln -s /opt/freeware/bin/pip /usr/bin/pip
 ```
 
+#### tap2junit
+
+```bash
+pip install tap2junit
```

Comment:
```suggestion
python -m pip install --upgrade pip pipenv git+https://github.com/refack/tap2junit.git
```

Reply:
Why not https://github.com/jbergstroem/tap2junit? (Or perhaps tap2junit should be moved under the foundation?)

Reply:
I made a few bug fixes https://github.com/refack/tap2junit/commit/7d9574915148b488b23be699f2a97f6eba509160

Reply:
https://github.com/jbergstroem/tap2junit/commit/7d9574915148b488b23be699f2a97f6eba509160